### PR TITLE
[charts/newrelic-pixie] support for nodeSelector and pod affinity

### PR DESCRIPTION
**Pull request by smcavallo**
Signed-off-by: smcavallo <smcavallo@hotmail.com>

#### Is this a new chart
No

#### What this PR does / why we need it:
Pixie currently does not run on windows nodes.  Add support for affinity and tolerations to limit deployment to supported node types.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes https://github.com/newrelic/helm-charts/issues/501

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

**Comment by jdfalk** [comment]
Fixes #501

File: charts/newrelic-pixie/README.md [hunk 1]
```diff
@@ -21,7 +21,9 @@ By default, Pixie is installed in the `pl` namespace.
...
+| `nodeSelector`                                             | Node label to use for scheduling                                                                                                                                                                                                      | `{}`                                   |
+| `tolerations`                                              | List of node taints to tolerate (requires Kubernetes >= 1.6)                                                                                                                                                                          | `[]`                                   |
+| `affinity`                                                 | Node affinity to use for scheduling                                                                                                                                                                                            | `[]`                  |
```
Comment (gsanchezgavier) [inline charts/newrelic-pixie/README.md:26]:
```suggestion
| `affinity`                                                 | Node affinity to use for scheduling                                                                                                                                                                                            | `{}`                  |
```

**Review by gsanchezgavier — COMMENTED** [review]
Thanks for the contribution!

**Comment by smcavallo** [comment]
> 
> 
> Thanks for the contribution!

@gsanchezgavier - ready for re-review, thanks!

**Review by gsanchezgavier — APPROVED** [review]

**Pull request merged**